### ISSUE TITLE
Improve Passport.js user experience

### ISSUE THREAD

**Issue by willow9886** [GluuFederation/oxTrust#557]
Currently all strategy fields are blank text fields. It seems like we could benefit by providing some drop down options for the primary strategies, while still providing the option to choose `other` and fill in your own values. 

So for instance:

"Add new strategy" --> new drop down field appears --> admin chooses one of the supported strategies --> two new drop downs appear already pre-specified as "clientID" and "client secret". 

Basically we should give people the ability to customize to their desire, but the user experience could probably benefit from making it easy to accomplish the primary use case which is social login with large consumer IDPs like Github, FB, twitter, etc.

**Comment by syntrydy** [comment]
@willow9886, I think this issue https://github.com/GluuFederation/oxTrust/issues/1020 cover this already.

**Comment by willow9886** [comment]
@syntrydy looks good.
